### PR TITLE
Color Changed

### PR DESCRIPTION
**Pull request by hariharan-mahadevan-10884**
Changed to Light theme
Changed Radio button styles and added style to submit button

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -147,7 +147,7 @@ html, body {
 }
 
 .main {
-  border: 2px solid;
+  border: none;
```
Comment (sivaraam) [inline css/main.css]: Remove the border property.

File: css/light-styles.css [hunk 2]
```diff
@@ -4,23 +4,23 @@
    ========================================================================== */
 @import url('http://fonts.googleapis.com/css?family=Lato');
 html, body {
-	font-family: 'Lato', sans-serif;
-  background-color: #22222;
+	font-family: "source-sans-pro","Helvetica Neue",Helvetica,Arial,sans-serif;
```
Comment (sivaraam) [inline css/light-styles.css]: This font choice. Leave that as it was before.
screenshot from 2017-04-23 20-24-44
screenshot from 2017-04-23 20-24-54

File: css/main.css [hunk 3]
```diff
@@ -177,25 +177,25 @@ html, body {
...
+  //display: none;
 }
 
 .item {
   padding: 0.25em 0em;
 }
 
 table {
-  border:1px solid;
+  border:none;
```
Comment (sivaraam) [inline css/main.css]: Don't remove table borders.

File: css/main.css [hunk 4]
```diff
@@ -177,25 +177,25 @@ html, body {
...
   border-collapse: collapse;
   width: 100%
 }
 
-td, th {
-  border: 1px solid;
+table th, table tr , table td{
+  border:none;
```
Comment (sivaraam) [inline css/main.css]: Don't remove this either.

**Review by sivaraam — CHANGES_REQUESTED** [review]
Let the label text of form-items be as it was before.

File: css/main.css (hunk 3 above)
Comment (hariharan-mahadevan-10884) [reply css/main.css]: So what shall I do now?
You want the borders to be displayed?

**Comment by sivaraam** [comment]
Closing in favour of new PR

**Pull request closed without merging**